### ISSUE TITLE
Ability to get first and last segment?

### ISSUE THREAD

**Issue by bmitch** [spatie/url#14]
Really enjoying using this package.

Was wondering if you guys were open to the ability of retrieving the first and last segment of the URL?

Something like:

```
    /** @test */
    public function it_can_return_the_last_path_segment()
    {
        $url = Url::create()->withPath('opensource/php');
        $this->assertEquals('php', $url->getLastSegment());
    }

    /** @test */
    public function it_can_return_the_first_path_segment()
    {
        $url = Url::create()->withPath('opensource/php');
        $this->assertEquals('opensource', $url->getFirstSegment());
    }
```

If so I wouldn't mind submitting a PR for it.

**Comment by bmitch** [comment]
Put together a pull request: https://github.com/spatie/url/pull/15

**Comment by sebastiandedeyne** [comment]
Released in `v1.1.0`

**Comment by bmitch** [comment]
Great thanks!
